openapi/generator_test.go

```diff
@@ -75,6 +79,8 @@ type (
 func (*X) TypeName() string { return "XXX" }
 func (*W) Format() string   { return "wallet" }
 func (*W) Type() string     { return "string" }
+func (ns) Nullable() bool   { return true }
+func (ni) Nullable() bool   { return false }
```

Comment:
Need to add a test for a pointer field that implements the `Nullable` interface whos return `false` is still marked as nullable in the spec.

Reply:
`ni` is a pointer field (https://github.com/wI2L/fizz/pull/90/files/7117e5ded046fda57a5bfed5f4d9dba39bfc7de2..2926bb1465518a3ac0752de28ab81608e0b87d5b#diff-1aeb94cb3e94f23e1ef8738104e892722175c50655af95f3c562efc596a89b08R56)
with Nullable returning `false` and it's returning true...
Or am I missing something?

Reply:
My bad, was looking at the type definition, not the field declaration.